build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.6.1'
+    classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
Was this missed in #2439?

Reply:
This was accidentally left when I was locally checking the Kotlin upgrade, we want to keep 3.6 for AS / Canary as per [this comment](https://github.com/oppia/oppia-android/pull/2439#discussion_r555106934), and Kotlin 1.4 should still work.

Reply:
Done, moved it back to 3.6.1